### ISSUE TITLE
ReadRssiRequest potentially hang the request queue by not responding

### ISSUE THREAD

**Issue by rickymohk** [NordicSemiconductor/Android-BLE-Library#114]
My app polls the RSSI of connected devices by calling 
```
handler.postDelayed(new Runnable() {
    @Override
    public void run() {
        readRssi().with(mCallbacks).enqueue();
     }
},2000);
```
within `onRssiRead` in `mCallbacks`.

Occasionally, the `ReadRssiRequest` does not give a response and hence any subsequent requests cannot be dequeued. What can we do to prevent it hanging the request queue?

**Comment by philips77** [comment]
Hmm... Interesting. Perhaps a timeout could be added to this request, like there's for Connect Request.

**Comment by philips77** [comment]
Fixed in 2.2.1.
